wallet/src/service/Offers.ts

```diff
@@ -41,6 +44,41 @@ type GiveOrWantEntries = {
   };
 };
 
+const CapDataShape = harden({ body: M.string(), slots: M.array() });
```

Comment:
This looks right but is a concern of Endo. If it's not available in Endo please file an issue for Endo to provide it.